flutter/lib/src/sentry_flutter.dart

```diff
@@ -131,7 +131,11 @@ mixin SentryFlutter {
       integrations.add(NativeAppStartIntegration(
         SentryNative(),
         () {
-          return SchedulerBinding.instance;
+          try {
+            /// Flutter >= 2.12 throws if SchedulerBinding.instance isn't initialized.
+            return SchedulerBinding.instance;
```

Comment:
Same as in https://github.com/getsentry/sentry-dart/pull/801, `WidgetsFlutterBinding` implements `SchedulerBinding`. Therefor, you could use [`WidgetsFlutterBinding.ensureInitialized()`](https://api.flutter.dev/flutter/widgets/WidgetsFlutterBinding/ensureInitialized.html) or `WidgetsFlutterBinding.instance`. 

But at this point, the instance should already be initialized, so in theory this try catch shouldn't be needed?

Reply:
This method should be called in a specific order, See `WidgetsFlutterBindingIntegration`, so I'd rather only use the `instance` if already there otherwise this might trigger some other problems.